src/dlls/dbgshim/dbgshim.cpp

```diff
@@ -387,25 +387,63 @@ class RuntimeStartupHelper
         return hr;
     }
 
+    bool NoNullHandles(HANDLE *handleArray, DWORD arrayLength)
+    {
+        for (int i = 0; i < (int)arrayLength; i++)
+        {
+            HANDLE h = handleArray[i];
+            if (h == NULL)
```

Comment:
I am not sure, but the comment says that it could. Fix the comment?

```
            // fill in event handle -- if GetContinueStartupEvent fails, it will still return 
            // INVALID_HANDLE_VALUE in hContinueStartupEvent, which is what we want.  we don't
            // want to bail out of the enumeration altogether if we can't get an event from
            // one telesto.

            HANDLE hContinueStartupEvent = INVALID_HANDLE_VALUE;
            HRESULT hr = GetContinueStartupEvent(debuggeePID, pStringArray[idx], &hContinueStartupEvent);
            _ASSERTE(SUCCEEDED(hr) == (hContinueStartupEvent != INVALID_HANDLE_VALUE));

            pEventArray[idx] = hContinueStartupEvent;
```